server/docker/update-env.sh

```diff
@@ -3,16 +3,21 @@
 # This scripts create a '.env' file that is used for docker & docker-compose as an input of environment variables.
 # This script is called by gradle and get the current project version as an input param
 
-if [ $# -lt 1 ]; then
-  echo "USAGE: $0 <VERSION> <DEBUG> <SMOKE_TEST>"
+if [ $# -lt 2 ]; then
+  # <PROJECT_BUILD_ROOT> and <VERSION> are required!
+  echo "USAGE: $0 <PROJECT_BUILD_ROOT> <VERSION> [DEBUG] [SMOKE_TEST]"
```

Comment:
I wonder why we need to specify location.
The gradle build should be copying any "source" `.env` file into the build folder, and this file should only _ever_ modify a `.env` file that is _within_ the build folder...
If the "working" folder for all processes is always the gradle build folder (it should be), then we wouldn't need to specify the build root here...

Reply:
Now when we copy the `docker` dir into the project `build` dir, and executing the shell scripts from there, this is no longer needed.